kickoff/model.py

```diff
@@ -18,6 +19,8 @@ class Release(object):
     submitter = db.Column(db.String(250), nullable=False)
     _submittedAt = db.Column('submittedAt', db.DateTime(pytz.utc),
                              nullable=False, default=datetime.utcnow)
+    _shippedAt = db.Column('shippedAt', db.DateTime(pytz.utc),
+                             nullable=False, default=datetime.utcnow)
```

Comment:
Are you sure defaulting this to utcnow will work? I think this means that shippedAt will end up getting set to the current utc datetime when the release is _submitted_, which I don't think you want. I think you need this to default to NULL and update it when the "Shipped!" button is pressed.